Automattic/wp-calypso
# Sign Up: Social login first mobile only experiment

#### Changes proposed in this Pull Request

More experiement details in pbxNRc-1a4-p2
Proposal: p58i-bsj-p2



<table>
<tr>
<td>

<div>Default:</div>

<img width="300" alt="Screen Shot 2022-02-16 at 4 49 43 PM" src="https://user-images.githubusercontent.com/115071/154382815-588eadc2-1c84-482e-ac93-684e9535749d.png">


</td>
<td>
<div>Treatment:</div>
<img width="300" alt="Screen Shot 2022-02-17 at 12 31 14 PM" src="https://user-images.githubusercontent.com/115071/154565717-c6ad461f-762f-44a0-965d-80e08102cd45.png">


</td>


</tr>
</table>



#### Testing instructions
Point the public-api.wordpress.com to point to your sandbox. (this makes the assignment happen while the experiment is in staging) 

Set the annon user to treatment. 
Notice that you are able to sign up via the flow. 

Start again
Set the annon user to the control flow.
Notice that you are able to sign up via the flow as expected.
